src/main/java/com/google/api/gax/bundling/ThresholdBundleReceiver.java

```diff
@@ -29,18 +29,20 @@
  */
 package com.google.api.gax.bundling;
 
+import com.google.api.gax.core.ApiFuture;
+
 /**
  * Interface representing an object that receives bundles from a ThresholdBundler and takes action
- * on them.
+ * on them. Implementations of ThresholdBundleReceiver are guaranteed to be thread-safe.
```

Comment:
This guarantee can't be made from an interface. I think the wording needs to be "Implementations... should be thread-safe."